2016-06-03.md

```diff
@@ -0,0 +1,28 @@
+# SUBJECT
+
+## Multiple language support
+
+### Ramping up Rust
+* Rust contrib?
+
+### Backing away from Python
+Ever since our announcement of Python on Tessel 2 last year, we've been waiting eagerly for a crowd of Pythonistas to charge in and excitedly exclaim, "I want that! I want to make that happen!"
+
+However, we haven't seen it. The basic pathway for Python on Tessel 2 has been proven out, but we haven't heard from anyone who's interested in taking on full support - or even much interest for using it. So we're letting it go. You'll see it disappear from the homepage soon. We can always pick up [from here](https://github.com/tessel/tessel-python) if the enthusiasm arrives.
+
+### Language deployment plugins
+
+## Recent updates
+* What the last couple of releases have done
+* Intro Team Members since the last time we did that
+
+## Upcoming conferences
```

Comment:
Nothing Tessel-specific but I will be at [NodebotsNYC](http://nodebots.nyc) next Tuesday to do some Tessel development. :smile: